### PR TITLE
Update overview.md

### PR DESCRIPTION
**Pull request by claycrenshaw**
Style edits.

### CODE REVIEW

**Comment by claycrenshaw** [comment]
I don't know what you mean by "avoid using forceful verbs"

**Comment by joshblack** [comment]
@claycrenshaw if you run it through that hemingway app thing it'll flag it, basically just tries to keep adverb usage down.

screen shot 2018-08-21 at 2 57 42 pm


A tangible example might be:

> Practical advice for ensuring that your sentences are alive:
> Use forceful verbs—replace long verb phrases with a more specific verb. For example, replace “She argues for the importance of the idea” with “She defends the idea.”

[Source](https://writingcenter.unc.edu/tips-and-tools/revising-drafts/)

https://english.stackexchange.com/questions/237053/are-adverbs-frowned-upon-in-proper-english-academic-writing also has a good answer here for adverb usage.

Definitely not a hard-and-fast rule, nor may it apply to the specific example above, just was trying to flag some of the things that the checker initially came back with 😄 If it makes sense, then go for it 👍

**Comment by joshblack** [comment]
Totally goofed btw, meant to say use forceful verbs 😄

**Comment by claycrenshaw** [comment]
Cool, thanks for the tip on Hemingway! Sometimes I disagree with him. /shrug

File: src/content/guidelines/accessibility/overview.md [hunk 1]
```diff
@@ -1,48 +1,142 @@
...
 
 ## Main Categories of Disabilities
 
-Quick jump
-- [Blind](/guidelines/accessibility/disabilities/#blind)
-- [Low Vision](/guidelines/accessibility/disabilities/#low-vision)
-- [Deaf and Hard of Hearing](/guidelines/accessibility/disabilities/#deaf)
-- [Physical Disabilities](/guidelines/accessibility/disabilities/#physical)
-- [Color Blindness](/guidelines/accessibility/disabilities/#color)
-- [Cognitive Disabilities](/guidelines/accessibility/disabilities/#cognitive)
```
Comment (alisonjoseph) [inline src/content/guidelines/accessibility/overview.md]: Overall looks good, was just wondering if we should leave this quick jump menu in so users can jump down to the specific sections?

Comment (claycrenshaw) [reply src/content/guidelines/accessibility/overview.md]: This was left over from when that content was on a separate page. This content isn't long enough to justify the shortcuts.

Comment (alisonjoseph) [reply src/content/guidelines/accessibility/overview.md]: gotcha :) makes sesne! merging in

**Pull request merged**